universum/modules/output/html_output.py

```diff
@@ -141,6 +141,9 @@ def _build_html_head():
             .exceptionTag {
                 color: darkred;
             }
+            .stderrTag {
+                color: orange;
```

Comment:
In my opinion this color is too bright for white background.
It is ok on a black background.

Reply:
As discussed in chat, it looks +/- okay. Let's leave it as is for now.